include/rviz_visual_tools/rviz_visual_tools.h

```diff
@@ -658,8 +658,8 @@ class RvizVisualTools
    * \param scale - size of axis
    * \return true on success
    */
-  bool publishAxisLabeled(const Eigen::Affine3d &pose, const std::string &label, const scales &scale = REGULAR);
-  bool publishAxisLabeled(const geometry_msgs::Pose &pose, const std::string &label, const scales &scale = REGULAR);
+  bool publishAxisLabeled(const Eigen::Affine3d &pose, const std::string &label, const colors &color = WHITE, const scales &scale = SMALL);
+  bool publishAxisLabeled(const geometry_msgs::Pose &pose, const std::string &label, const colors &color = WHITE, const scales &scale = SMALL);
```

Comment:
could you switch this to have scale first, then color? that way it maintains backwards compatibility